### ISSUE TITLE
Bug: Controller Detected On Boot Without A Controller Connected

### ISSUE THREAD

**Issue by wyatt-webb** [shawn-lindberg/sandman#19]
On the boot of Sandman it announces that a controller is connected even though the only two devices connected to the Raspberry Pi via USB are the speaker and USB audio adapter.

**Comment by shawn-lindberg** [comment]
Yes, I think this would be good to fix. There is a default device in the config file, but it may not necessarily be a controller. The default device should probably be removed and we can try to handle this better once there is a better way to modify the config file.
